node/src/GlideClusterClient.ts

```diff
@@ -612,10 +610,9 @@ export class GlideClusterClient extends BaseClient {
         message: string,
         route?: Routes,
     ): Promise<ClusterResponse<string>> {
-        return this.createWritePromise(
-            createEcho(message),
-            toProtobufRoute(route),
-        );
+        return this.createWritePromise(createEcho(message), {
+            route: toProtobufRoute(route),
```

Comment:
decoder?

Reply:
it should be added to all commands which returns string. and it will be gradually.